### PR TITLE
Improve documentation on activity module

### PR DESCRIPTION
**Pull request by godiard**

### CODE REVIEW

**Comment by godiard** [comment]
This is a initial test of the new docs format

**Comment by godiard** [comment]
Reference http://wiki.sugarlabs.org/go/Features/Sugar3Docs

File: src/sugar3/activity/activity.py [hunk 1]
```diff
@@ -19,6 +21,9 @@ class ReadActivity(activity.Activity):
         pass
 
 To get a real, working activity, you will at least have to implement:
+
+.. code-block:: python
+
     __init__(), read_file() and write_file()
```
Comment (samdroid-apps) [inline src/sugar3/activity/activity.py]: Why do we need to make this a code block?  Could we make these items links and part of the paragraph?  :func:`sugar3.activity.activity.Activity.__init__`

Comment (godiard) [inline src/sugar3/activity/activity.py]: ok

**Pull request closed without merging**